vnext/ReactUWP/Base/CoreNativeModules.cpp

```diff
@@ -117,7 +115,7 @@ std::vector<facebook::react::NativeModuleDescription> GetCoreModules(
         if (HasPackageIdentity()) {
           return std::make_unique<facebook::react::AsyncStorageModule>(L"asyncStorage");
         } else {
-          return std::make_unique<facebook::react::AsyncStorageModuleWin32>();
+          return nullptr;
```

Comment:
>return nullptr; [](start = 10, length = 15)

We cannot do it. Adding a nullptr for the module will crash all users of XAML islands.

Reply:
Worked around it. Reverting.